tex/rules.tex

```diff
@@ -363,14 +363,23 @@ \subsection{Damaged robots \label{ref-012}}
 Only the referee decides whether a robot is damaged. A robot can only be taken
 off or returned with the referee's permission.
 
+\replaced[id=TC]
+{
+If both robots from the same team are deemed damaged at kickoff,
+gameplay will be paused and the remaining team will be awarded 1 goal for
```

Comment:
@gngrbrdmn I changed "point" to "goal" on this line for consistency with the bigger file -- hopefully you are OK with that.